tests/lib/in_memory/test_database.py

```diff
@@ -94,10 +94,12 @@ def test_patient_column_binary_op_patient_column():
         4 | 411
         """
     )
-    assert pc1.binary_op(add, pc2) == Column.parse(
+    assert pc1.binary_op_with_null(add, pc2) == Column.parse(
```

Comment:
Could we have a test that shows what `.binary_op()` does?

Reply:
This has been reverted.